### DISCUSSION TITLE
Form Validation question when using [] in the name of the field

### DISCUSSION THREAD

**Discussion by princeandrew01** [fomantic/Fomantic-UI#2032] [Behaviors (Api, Form Validation, Visibility)] [answered]
Good Day,

Just had a general question, I have a few input fields that share the same name example: easy[], the reason I am using this is when they get submitted they basically appear as an array and I can process it by just looping through each element.

I tried adding validation like this:
`	$('#main_form').form({
			fields: {
			easy: {
					identifier: 'easy[]',
					rules: [
						{
							type: 'integer',
							prompt: 'Must be a number'
						}
					]
				}
			}
		})`

However it is only validating the first field it comes across. Is there anyway for it to validate all the fields that match easy[] instead of just the first?

**Comment by lubber-de** [comment, marked as answer]
Works as expected for me
See https://jsfiddle.net/lubber/1q3dh6pf/5/

**Reply by princeandrew01** [reply]
Wonder what I am doing wrong, only big difference is I am running one version behind. Will try again, thanks for the feedback.

Edit: Just realized there is something I did not explain its in a table and I dynamically adding rows to the table with javascript with a + button. I'll modify the jsfiddle you sent as an example and post it a bit later.

**Reply by lubber-de** [reply]
You may try `$('.ui.form').form('refresh')` after adding new fields to the form

**Reply by lubber-de** [reply]
Maybe the fix from #1811 is the other reason why it does not work for you using 2.8.7 otherwise

**Reply by princeandrew01** [reply]
the `$('.ui.form').form('refresh')` worked! Thanks a bunch
